### DISCUSSION TITLE
non-transformed data

### DISCUSSION THREAD

**Discussion by alexZhongvai** [whitews/FlowKit#134] [General]
First of all, thanks for the wonderful package, with which I can convert flow data to pandas dataframe and makes my life a lot easier.

I just wonder how to extract non-transformed data when I use the following command to extract fj workspace that has transformation done?  I plugged `ignore_transformation =True` in, but nothing happened. 

# import a FlowJo 10 workspace file
`session.import_flowjo_workspace(wsp_path)`

**Comment by whitews** [comment]
Hi Alex,

Sorry I missed this message. You can use the `get_gate_membership` method to retrieve a Boolean array that has True values for Sample events that are in the gate. With that Boolean array you can filter the Sample events (retrieved via the Sample `get_events` method).

Also, note that the upcoming 1.0 version will have breaking API changes for handling FlowJo workspace files. Basically there will be a dedicated Workspace class that will hopefully make things simpler for users.

Regards,
Scott

**Reply by alexZhongvai** [reply]
Thanks so much, Scott. I am looking forward to the new version!!
